examples/hif2a/generate_star_map.py

```diff
@@ -39,297 +38,183 @@
 # 2. Identify ligand subset that shares a common substructure
 
 # SMARTS courtesy of YTZ (Jan 26, 2021)
-eyeballed_smarts_pattern = '[*]~1~[*]~[*]~2~[*]~[*]~[*]~[*](~[#8]~[*]~3~[*]~[*]~[*]~[*]~[*]~3)~[*]~2~[*]~1'
-bicyclic_query_mol = Chem.MolFromSmarts(eyeballed_smarts_pattern)
+bicyclic_smarts_pattern = '[*]~1~[*]~[*]~2~[*]~[*]~[*]~[*](~[#8]~[*]~3~[*]~[*]~[*]~[*]~[*]~3)~[*]~2~[*]~1'
+bicyclic_query_mol = Chem.MolFromSmarts(bicyclic_smarts_pattern)
 
 # filter matches
-mols_with_core_1 = [mol for mol in mols if mol.HasSubstructMatch(bicyclic_query_mol)]
+has_bicycle = lambda mol: len(mol.GetSubstructMatches(bicyclic_query_mol)) == 1
```

Comment:
shouldn't this be >= 1? It's possible for there to be duplicate matches

Reply:
No, should be == 1. (We specifically wanted to exclude the molecule in this set that matches this pattern twice) Will document

Reply:
<img width="868" alt="image" src="https://user-images.githubusercontent.com/5759036/110020711-e115ca00-7cf7-11eb-9c85-b5d498fc997c.png">

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/360/commits/67cd114fca7e7635e2d0ed2aa33c3616c098e2b2